README.md

```diff
@@ -23,12 +23,7 @@ import Ember from 'ember';
 import StateMixin from 'ember-state-services/mixin'
 
 export default Ember.Object.extend(StateMixin, {
-  stateName: 'emailEdit',
-  setupState: function(Factory, model) {
```

Comment:
I removed because I'm confused why we would them how to re-implement the default behavior?